TypeScript.YAML-tmLanguage

```diff
@@ -15,6 +15,7 @@ variables:
   propertyIdentifier: '\#?{{identifier}}'
   constantPropertyIdentifier: '\#?{{constantIdentifier}}'
   quotedStrings: (\'([^\'\\]|\\.)*\')|(\"([^\"\\]|\\.)*\")|(\`([^\`\\]|\\.)*\`)
+  aliasString: \'(?:[^\'\\]|\\.)*\'|\"(?:[^\"\\]|\\.)*\"
```

Comment:
You dont need this. just use quotedStrings instead. Grammar here can be more permissive than actual and thats ok.